File: java-single/app/src/main/java/javaSingle/gql/meters/MeterLabel.java
package javaSingle.gql.meters;

import javaSingle.klv.common.enums.MeterSourceOption;

public class MeterLabel {

  public final MeterSourceOption option;

  public final Integer row;

  public final Integer column;

  public final Integer height;

  public final Integer width;

  public MeterLabel(MeterSourceOption option, Integer row, Integer column, Integer height, Integer width) {
    this.option = option;
    this.row = row;
    this.column = column;
    this.height = height;
    this.width = width;
  }
}

File: rust-single/src/gql/meters/meterLabel.rs
use crate::klv::common::enums::meterSourceOption::MeterSourceOption;

#[derive(Debug, Clone, Copy)]
pub struct MeterLabel {
  pub option: MeterSourceOption,
  pub row: i32,
  pub column: i32,
  pub height: i32,
  pub width: i32,
}
